chrome/content/zotero/preferences/preferences_export.jsx

```diff
@@ -82,6 +85,65 @@ Zotero_Preferences.Export = {
 	}),
 	
 	
+	/*
+	 * Builds the note Quick Copy drop-down from the current global pref
+	 */
+	populateNoteQuickCopyList: async function () {
+		// Initialize default format drop-down
+		var format = Zotero.Prefs.get("export.noteQuickCopy.setting");
+		format = Zotero.QuickCopy.unserializeSetting(format);
+		var menulist = document.getElementById("zotero-noteQuickCopy-menu");
+		menulist.setAttribute('preference', "pref-noteQuickCopy-setting");
+		
+		if (!format) {
+			format = menulist.value;
+		}
+		
+		format = Zotero.QuickCopy.unserializeSetting(format);
+		
+		menulist.selectedItem = null;
+		menulist.removeAllItems();
+		
+		var popup = document.createElement('menupopup');
+		menulist.appendChild(popup);
+
+		// add export formats to list
+		var translation = new Zotero.Translate("export");
+		var translators = await translation.getTranslators();
+		
+		// Add/override Note Text translator
+		translators = translators.filter(x => x.translatorID != 'a45eca67-1ee8-45e5-b4c6-23fb8a852873');
+		translators.push({
+			translatorID: 'a45eca67-1ee8-45e5-b4c6-23fb8a852873',
+			label: 'Note Text',
+			configOptions: {
+				noteTranslator: true
+			}
+		});
```

Comment:
Now that Markdown is the text format, it doesn't really make sense to show that separately, so I think the visible options for Quick Copy should be "Markdown + Rich Text" and "HTML", in that order. No "Note" prefix. The "Rich Text" part should probably be localized.

The export dialog should have "Markdown" and "HTML", with no "Text". I don't think we need the "Note" prefixes there (and then we don't need to bother localizing them).

Reply:
Do you mean we should remove "Note" prefix from the actual translators?

Reply:
No, in the display.

Reply:
I agree that not having a separate Markdown option is simpler and better, although in some cases user might want Markdown even when dragging into a text box that supports Rich Text. But probably no need to worry about that.

Reply:
Yeah that occurred to me but I don't think it's necessary. Seems pretty unlikely, and anyone in that situation can probably figure out how to get the Markdown.